language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -47,63 +47,66 @@ optionsParserInfo = info (optionsParser <**> helper)
     <> progDesc "Generate TypeScript bindings from a DAR"
     )
 
+-- Each package has a unique ID and is perhaps ascribed a symbolic
+-- name (that is, if a given package has been determined to be the
+-- "main-package" of some DAR).
+type PackageMap = Map.Map PackageId (Maybe String, Package)
```

Comment:
I find these kind of type synonyms to be more confusing than helpful. Either turn it into a newtype or inline it.

Reply:
Ok